schemas/system/SignUp.schema

```diff
@@ -0,0 +1,15 @@
+Form({
+  Login: { domain: 'Login', required: true },
+  Password1: { domain: 'PasswordHash', required: true },
```

Comment:
IIUC this would be just a string, it will become hash when we save a password to a db.

Reply:
I'd prefer to name field `FirstPassword` and `SecondPassword` accordingly.

Reply:
We may use SHA-512
@belochub ?

Reply:
I think domain should be called something like `PlainTextPassword`.